### ISSUE TITLE
English page detected as Portuguese

### ISSUE THREAD

**Issue by Jamie-99a** [validator/validator#2020]
### URL for document that reproduces the bad-language-detection problem

https://www.travelvictoria.com.au/victoria/government/regionalcouncils/

### Document upload

This warning is coming up for the page which is written completely in English:

info warning: This document appears to be written in Portuguese but the “html” start tag has “lang="en"”. Consider using “lang="pt"” (or variant) instead.

### Raw source / text input

```html
n/a
```

**Comment by sideshowbarker** [comment]
Thank for taking time to report this. It’s led to me making the language detector slightly smarter (I hope).

I’ve deployed a fix for it — so if you try again now:

https://validator.w3.org/nu/?doc=https://www.travelvictoria.com.au/victoria/government/regionalcouncils/

… I think you should find that the checker is no longer misidentifying the language of the page.
